File: app/src/main/java/com/ninhttd/devtest/data/entity/SeoInfo.java

package com.ninhttd.devtest.data.entity;

import com.google.gson.annotations.SerializedName;

@SuppressWarnings("unused")
public class SeoInfo {

    @SerializedName("description")
    private String mDescription;
    @SerializedName("metaDescription")
    private String mMetaDescription;
    @SerializedName("metaKeyword")
    private String mMetaKeyword;
    @SerializedName("metaTitle")
    private String mMetaTitle;
    @SerializedName("shortDescription")
    private String mShortDescription;

    public String getDescription() {
        return mDescription;
    }

    public void setDescription(String description) {
        mDescription = description;
    }

    public String getMetaDescription() {
        return mMetaDescription;
    }

    public void setMetaDescription(String metaDescription) {
        mMetaDescription = metaDescription;
    }

    public String getMetaKeyword() {
        return mMetaKeyword;
    }

    public void setMetaKeyword(String metaKeyword) {
        mMetaKeyword = metaKeyword;
    }

    public String getMetaTitle() {
        return mMetaTitle;
    }

    public void setMetaTitle(String metaTitle) {
        mMetaTitle = metaTitle;
    }

    public String getShortDescription() {
        return mShortDescription;
    }

    public void setShortDescription(String shortDescription) {
        mShortDescription = shortDescription;
    }

}

File: app/src/main/java/com/ninhttd/devtest/data/entity/SeoInfo.kt
package com.ninhttd.devtest.data.entity

import com.google.gson.annotations.SerializedName

class SeoInfo {

    @SerializedName("description")
    var description: String? = null
    @SerializedName("metaDescription")
    var metaDescription: String? = null
    @SerializedName("metaKeyword")
    var metaKeyword: String? = null
    @SerializedName("metaTitle")
    var metaTitle: String? = null
    @SerializedName("shortDescription")
    var shortDescription: String? = null

}
